src/components/GuestList/GuestList.tsx

```diff
@@ -8,14 +8,40 @@ import { StyledButton, StyledCard } from '../App/styledComponents.styles'
 type NewGuest = {
   id: number,
   guestName: string,
-  phoneNumber: number
+  phoneNumber: string;
 }
 
 const GuestList: React.FC = () => {
+ 
+  const [guestName, setGuestName] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [guests, setGuests] = useState<NewGuest[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
 
-  const [guestName, setGuestName] = useState('')
-  const [phoneNumber, setPhoneNumber] = useState(0)
-  const [guests, setGuests] = useState<NewGuest[]>([])
+  const checkNumber = (event: any) => {
+    let value = event.trim().replaceAll( "-", "")
+    var reg = /^\d+$/;
+    if (reg.test(value)) {
+      formatPhoneText(value)
+      setIsError(false)
+      setErrorMessage('')
+    } else {
+      setIsError(true)
+      setErrorMessage('Phone number only accepts numerical values')
+    }
+  } 
+
+  const formatPhoneText = (value: string) => {
+    if (value.length > 3 && value.length <= 6) {
+      value = value.slice(0,3) + "-" + value.slice(3);
+    } else if(value.length > 6) {
+      value = value.slice(0,3) + "-" + value.slice(3,6) + "-" + value.slice(6);
+    }
+    setPhoneNumber(value)
```

Comment:
We might need to add a "+1" to the front of all the phone numbers for Twilio, but we will see when we get to that point